kentikapi/internal/validation/validate.go

```diff
@@ -83,3 +85,81 @@ func validate(method string, direction string, path string, v reflect.Value) []s
 	}
 	return missing
 }
+
+// ValidateCECreateRequest checks if CloudExport create request contains all required fields.
+//nolint:gocyclo
+func ValidateCECreateRequest(ce *models.CloudExport) error {
+	if ce == nil {
+		return nil
```

Comment:
```suggestion
		return fmt.Errorf("cloud export object is nil")
```
This should be an error, because there is no point in sending empty requests. Please also modify "nil request" subtest, of course.

Reply:
Done. Added separate comment for "nil request" subtest